dash-pipeline/SAI/templates/utils.cpp.j2

```diff
@@ -217,20 +217,77 @@ sai_status_t sai_create_switch_dummy(
     return SAI_STATUS_SUCCESS;
 }
 
-sai_status_t sai_get_switch_attribute_dummy(
+#define DASH_BMV2_NUM_PORTS 2 
```

Comment:
is this duplicate with pr #281

Reply:
> is this duplicate with pr #281 ?

Yes it is, see https://github.com/Azure/DASH/pull/278#discussion_r1020607888. Once 281 is merged, we'll sync this PR and some changes will disappear because they'll already be in main.